apps/prairielearn/src/lib/editors.ts

```
@@ -962,6 +963,135 @@
+
+    // Update the infoAssessment.json files to include the course sharing name for each question
+    await updateInfoAssessmentFiles(courseInstancePath, this.from_course.sharing_name);
+
+    debug('Read infoCourseInstance.json');
+    const infoJson = await fs.readJson(path.join(courseInstancePath, 'infoCourseInstance.json'));
+
+    debug('Write infoCourseInstance.json with new longName and uuid');
+    infoJson.longName = names.longName;
+    infoJson.uuid = this.uuid;
+    // We do not want to preserve sharing settings when copying an assessment to another course
+    delete infoJson['shareSourcePublicly'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)